dist/client.d.ts

```diff
@@ -0,0 +1,14 @@
+import { Connection } from "./connection";
+import { IConnection } from "./util/types/connectionTypes";
+export default class Client extends Connection {
```

Comment:
add one line break before this.

Reply:
@VikasPatel these are declaration files created via script do you still want me to add the line break